TestDB.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import sys; sys.path.append('../')
+import faker
+
+class TestUser(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.faker = faker.Faker()
+        faker = self.faker
+
+    def test_save_customer(self):
+        customer = Customer()
+        customer.first_name = faker.first_name()
+        customer.last_name = faker.last_name()
+        customer.date_created = faker.date()
+        user = User()
```

Comment:
We are now rethinking "User." I think we're going to strike it and focus our work on database interaction. We had all our methods on "User." It is probably more explicit to have them on the classes themselves. Thus, if we create a customer, we should simply create a customer rather than create a user to create a customer.